src/MUIDataTableBody.js

```diff
@@ -106,6 +106,8 @@ class MUIDataTableBody extends React.Component {
                     dataIndex: dataIndex,
                   })}
                   checked={this.isRowSelected(dataIndex)}
+                  fixedToolbar={false}
+                  fixedHeader={false}
```

Comment:
I had a propTypes warning here for the fixedToolbar and fixedHeader, so I decided to fix it. These are the SelectCells not in the header row.